modules/nf-core/gatk4/germlinecnvcaller/main.nf

```diff
@@ -0,0 +1,75 @@
+process GATK4_GERMLINECNVCALLER {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'broadinstitute/gatk:4.4.0.0':
+        'broadinstitute/gatk:4.4.0.0' }"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "GATK4_GERMLINECNVCALLER module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    tuple val(meta), path(tsv), path(intervals)
+    path model
+    path ploidy
+
+    output:
+    tuple val(meta), path("*-calls.tar.gz"), emit: calls, optional: true
+    tuple val(meta), path("*-model.tar.gz") , emit: model, optional: true
+    path  "versions.yml"             , emit: versions
```

Comment:
A little bit of nitpicking but it would look nicer if you could align the commas